src/tests/nativeaot/SmokeTests/UnitTests/Interfaces.cs

```diff
@@ -1020,4 +1021,77 @@ public static void Run()
             TestVariantInterface<GenericVariantWithHiddenDerived<Derived>, Mid>("GenericVariantWithHiddenBase.WhichMethod(Mid)");
         }
     }
+
+    class TestSimpleStaticDefaultInterfaceMethods
+    {
+        interface IFoo
+        {
+            static virtual string GetCookie() => nameof(IFoo);
```

Comment:
This is going to need https://github.com/dotnet/runtime/pull/69756/commits/3e7142334eb128c75b01da7885c11949ec9cc65a

That is you need `<MicrosoftNetCompilersToolsetVersion>4.3.0-2.22270.4</MicrosoftNetCompilersToolsetVersion>` or later